tests/multi_ptr/multi_ptr_local_accessor_constructor.h

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for multi_ptr local_accessor constructors
+//
+*******************************************************************************/
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_LOCAL_ACCESSOR_CONSTRUCTORS_H
+#define __SYCLCTS_TESTS_MULTI_PTR_LOCAL_ACCESSOR_CONSTRUCTORS_H
+
+#include "../common/common.h"
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_local_accessor_constructors {
+
+constexpr int expected_val = 42;
```

Comment:
This variable used `expected_val` just once, can we avoid using this variable?

Reply:
Removed and replaced its usage with integer literal.

Reply:
I think it's not a good idea. We don't get a lot of benefits after deleting the variable, but we get a "magic number" in our code. It's always hard to understand what this number does, especially after some time.

Reply:
I think you are right, returned the variable for better readability.